tests/test_csc.py

```diff
@@ -863,10 +873,9 @@ async def test_check_applied_forces_in_range(self) -> None:
         ):
             await salobj.set_summary_state(self.remote, salobj.State.ENABLED)
 
-            # Check we have the force data from simulator or not
-            await asyncio.sleep(SLEEP_TIME_SHORT)
-            self.assertTrue(self.csc.tel_axialForce.has_data)
-            self.assertTrue(self.csc.tel_tangentForce.has_data)
+            # Wait until we have the force data from simulator
+            await self.remote.tel_tangentForce.next(flush=False, timeout=STD_TIMEOUT)
```

Comment:
You should use await self.assert_next_sample(topic=self.remote.tel_tangentForce instead of next. This is because assert_next_sample has some handling for kafka.

Reply:
Fixed in the commit: [2600cdf](https://github.com/lsst-ts/ts_m2/pull/76/commits/2600cdfde9207c9f559e5d6492740258923fa2ba). Thanks!